src/App.js

```diff
@@ -1,23 +1,17 @@
-import logo from './logo.svg';
-import './App.css';
+//testTask
```

Comment:
No need to write this kind of comments.
You don't have to explain obvious things using comments.

You can write comments in 3 cases:
 - Documentation
 - some TODO or Refactoring tips. E.x. ```// TODO: clean up this function```
 - When you have a complex logic and you need to explain it.

In all other cases just don't write it. It's won't help you but can lead to misunderstanding.